### ISSUE TITLE
Armors do not load correctly in 1.21

### ISSUE THREAD

**Issue by bladeRR** [PluginBugs/Issues-ItemsAdder#3986]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I've looked for already existing issues on the [Issue Tracker](https://github.com/PluginBugs/Issues-ItemsAdder/issues) and haven't found any.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.
- [X] I tested this with just ItemsAdder and its dependencies and with a vanilla client of the same version as the Server.

### Discord Username (optional)

bladeRR1

### What happened?

I have the armor configured correctly, it looks like the icon is ok, but when I put it on my body it doesn't display correctly, I have no shaders or mods.
<img width="268" alt="knight" src="https://github.com/user-attachments/assets/29ffd089-6427-42ac-a397-275fa69d1424">

### Steps to reproduce the issue

when I set my server to 1.21 the armors stopped showing, in previous versions it worked correctly

### Server version

his server is running Paper version 1.21-130-master@b1b5d4c (2024-08-10T10:02:42Z) (Implementing API version 1.21-R0.1-SNAPSHOT)
You are running the latest version
Previous version: 1.21-2234-dd826a3 (MC: 1.21)

### ItemsAdder Version

[17:50:48 INFO]: ItemsAdder version 4.0.2-beta-release-9
[17:50:48 INFO]: Customized Minecraft without client mods!
[17:50:48 INFO]: Website: devs.beer
[17:50:48 INFO]: Author: LoneDev

### ProtocolLib Version

[17:51:14 INFO]: ProtocolLib version 5.3.0-SNAPSHOT-726
[17:51:14 INFO]: Provides read/write access to the Minecraft protocol.
[17:51:14 INFO]: Authors: dmulloy2 and comphenix

### LoneLibs Version

[17:51:24 INFO]: LoneLibs version 1.0.58
[17:51:24 INFO]: Collection of libs used by my plugins
[17:51:24 INFO]: Authors: LoneDev and others

### Full Server Log

https://mclo.gs/QXCtkCu

### Problematic items yml configuration file (optional)

```yaml
armors_rendering:
  king_arthur_animated_weapon_set:
    color: '#8c196b'
    layer_1: king_arthur_animated_weapon_set/king_arthur_armor_layer_1
    layer_2: king_arthur_animated_weapon_set/king_arthur_armor_layer_2
    use_color: false
items:
  king_arthur_animated_weapon_set_chestplate:
    display_name: ᴘᴇᴄʜᴇʀᴀ ᴅᴇʟ ʀᴇʏ ᴀʀᴛᴜʀᴏ
    resource:
      generate: true
      textures:
      - king_arthur_animated_weapon_set/king_arthur_chestplate.png
    specific_properties:
      armor:
        slot: chest
        custom_armor: king_arthur_animated_weapon_set
  king_arthur_animated_weapon_set_leggings:
    display_name: ᴍᴀʟʟᴀs ᴅᴇʟ ʀᴇʏ ᴀʀᴛᴜʀᴏ
    resource:
      generate: true
      textures:
      - king_arthur_animated_weapon_set/king_arthur_leggings.png
    specific_properties:
      armor:
        slot: legs
        custom_armor: king_arthur_animated_weapon_set
```

### Screenshots/Videos (you can drag and drop files or paste links)

<img width="268" alt="knight" src="https://github.com/user-attachments/assets/4ab2fe88-50f9-4944-8a93-e1e7a0a68e63">

**Comment by Andre601** [comment]
Please provide an actual latest.log as the one you have is not a valid log at all.

**Comment by idclmfao** [comment]
I have this issue as well

**Comment by bladeRR** [comment]
> Please provide an actual latest.log as the one you have is not a valid log at all.

[latest.log](https://github.com/user-attachments/files/16899556/latest.log)

https://mclo.gs/RHqoQ46

please do not listen to mm because mcpets is needed.

**Comment by Andre601** [comment]
Please test if this issue also exists with just ItemsAdder and its dependencies installed, to eliminate the possability of another plugin messing things up, given that there are a ton of warnings and issues from various other plugins...

**Comment by bladeRR** [comment]
> Pruebe si este problema también existe con solo ItemsAdder y sus dependencias instaladas, para eliminar la posibilidad de que otro complemento arruine las cosas, dado que hay un montón de advertencias y problemas de varios otros complementos...

https://mclo.gs/m1bcfuS

<img width="173" alt="error" src="https://github.com/user-attachments/assets/3c388d13-5c7f-4811-b194-e4a3ef5df2a4">


no, the error persists

**Comment by JrRequeim** [comment]
Have the same issue, armor textures can't be loaded correctly :/

**Comment by Skippy8787** [comment]
Also Having the exact same issue with armour's not loading and showing as dyed leather armour. 
No Errors to present.

**Comment by LiquidBoosts** [comment]
I have the same issue

**Comment by naikyys** [comment]
Same error..

**Comment by Barlaf** [comment]
Hello ! Same probleme with 1.21.1
```
This server is running UniverseSpigot version 1.21.1-DEV-ver/1.21.1@014ae00 (2024-09-08T01:48:53Z) (Implementing API version 1.21.1-R0.1-SNAPSHOT)
Universe-1.21.1-DEV-014ae00 (MC: 1.21.1)

ItemsAdder: 4.0.2-beta-release-10
LoneLibs: 1.0.58
```
Sample config:
```
  ares_chestplate:
    display_name: '&#ff990f&lA&#fd9c0f&lr&#fc9f0f&lm&#faa20f&lu&#f9a510&lr&#f7a810&le &#f6aa10&ld&#f4ad10&le &#f3b010&ll&#f1b310&la &#f0b611&lg&#eeb911&la&#ecbc11&lr&#ebbf11&ld&#e9c211&le &#e8c511&lr&#e6c711&lo&#e5ca12&ly&#e3cd12&la&#e2d012&ll&#e0d312&le'
    resource:
      generate: true
      textures: 
      - enderkariz/ares_set/ares_chestplate
      model_id: 4211
    specific_properties:
      armor:
        slot: chest
        custom_armor: ares
    attribute_modifiers:
      legs:
        armor: 8
        armorToughness: 3
        knockbackResistance: 1
    durability:
      max_custom_durability: 1200
```
```
{"parent":"minecraft:item/generated","textures":{"layer0":"lunar_studios:enderkariz/ares_set/ares_chestplate","layer1":"lunar_studios:enderkariz/ares_set/ares_chestplate"}}
```
![image](https://github.com/user-attachments/assets/2d48557c-da9a-4281-8f56-a56e8e1a7c78)

**Comment by Rouxshi** [comment]
Hi guys, I just got the same problem yesterday  BUT, there's a solution according to a pinned message on discord; just go to the items adder config, look for the "generate-custom-armors-textures:", and DISABLE the "vanilla_17", then enable the "vanilla" one, then just /iareload /iazip.
You may want to make a backup because there's a chance of it messing with your resourcepack; anyway, after that the custom armor rendering works flawlessly.
You can check the pinned messages on the IA discord to see it yourself!

**Comment by Barlaf** [comment]
Oh @Rouxshi, how i marry you ?
All is perfect with this edit of conf file ! And i'm not the buyer of the plugin, i don't have acces to the Discord :(
Ty for helping

**Comment by SlyGuy89** [comment]
> Hi guys, I just got the same problem yesterday BUT, there's a solution according to a pinned message on discord; just go to the items adder config, look for the "generate-custom-armors-textures:", and DISABLE the "vanilla_17", then enable the "vanilla" one, then just /iareload /iazip. You may want to make a backup because there's a chance of it messing with your resourcepack; anyway, after that the custom armor rendering works flawlessly. You can check the pinned messages on the IA discord to see it yourself!

Can confirm this resolves the issue for Paper 1.21.1, thanks for sharing! :)

**Comment by bladeRR** [comment]
the solution is in the discord pin for servers in 1.21 onwards
